wsmux/mux.go

```diff
@@ -0,0 +1,15 @@
+package wsmux
+
+import (
+	"github.com/gorilla/websocket"
+)
+
+// Server (NOTE: set server field to 0 for server connection)
+func Server(conn *websocket.Conn, onRemoteClose func(*Session)) *Session {
+	return newSession(conn, 0, onRemoteClose)
```

Comment:
Let's not make it an `onRemoteClose`... let's instead define a config structure and take that...
Also `func()` should be sufficient, it's more likely to be useful...

```go
type Config struct {
  RemoteCloseCallback func()
}

func Server(conn *websocket.Conn, config Config) *Session {
}
```

This way we can add more config options later without breaking the API :)

NOTE: obviously all config options should be optional... Passing `nil` should be okay..